From 955758bb6408b8f249a87e3d795fae87939b7bae Mon Sep 17 00:00:00 2001
From: Ting Chou <ting.chou@sifive.com>
Date: Thu, 5 May 2022 14:28:49 +0800
Subject: [PATCH] [riscv64] Fix
 cctest/test-assembler-riscv64/RISCV_UTEST_FLOAT_WIDENING_vfwadd_vf.

Storing with E64 when SEW=32 has EMUL=2, which copies |n| 64 bit wide data to
the result double array already. Besides, it is illegal to access v1 when
EMUL=2.
---
 test/cctest/test-assembler-riscv64.cc | 2 --
 1 file changed, 2 deletions(-)

diff --git a/test/cctest/test-assembler-riscv64.cc b/test/cctest/test-assembler-riscv64.cc
index 0e18c468ed..55b4706300 100644
--- a/test/cctest/test-assembler-riscv64.cc
+++ b/test/cctest/test-assembler-riscv64.cc
@@ -2333,9 +2333,7 @@ UTEST_RVV_VF_VV_FORM_WITH_OP(vfdiv_vv, /)
       }                                                                        \
       __ instr_name(v0, v2, fa1);                                              \
       __ li(t1, Operand(int64_t(result)));                                     \
-      __ li(t2, Operand(int64_t(&result[n / 2])));                             \
       __ vs(v0, t1, 0, VSew::E64);                                             \
-      __ vs(v1, t2, 0, VSew::E64);                                             \
     };                                                                         \
     for (float rs1_fval : compiler::ValueHelper::GetVector<float>()) {         \
       for (float rs2_fval : compiler::ValueHelper::GetVector<float>()) {       \
-- 
2.25.1

